pages/home/self-hosting-overview.mdx

```diff
@@ -0,0 +1,16 @@
+---
+title: "Self-Hosting Overview"
+description: "Learn about the different ways to self-host Arcade"
+---
+
+# Self-Hosting Overview
+
+The best way to use Arcade is to use our cloud service.  However, if you need to run Arcade locally to either connect your tools to local-only resources (e.g. a local database or filesystem), or your policies require it, Arcade has you covered!
+
+## Hybrid Deployments
+
+Hybrid deployments combine the benefits of running Arcade locally with the benefits of using our cloud service.  You can use our cloud service to manage your users and authentication, but run a worker locally.
```

Comment:
Are we trying to remove "worker" or did I mishear that?

```suggestion
Hybrid deployments combine the benefits of running Arcade locally with the benefits of using our cloud service.  You can use our cloud service to manage your users and authentication, but run tools locally.
```

Reply:
Thanks! 9c912b7e3037b8e078ec4b3bb930c3e90897e3b6